progress/react/demo/README.md

```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Maybe write something for the demo readme

Reply:
Good catch, I've added a little blurb on how to run the project.